src/robusta/integrations/kubernetes/base_triggers.py

```diff
@@ -152,3 +169,30 @@ def build_execution_event(
             obj=obj,
             old_obj=old_obj,
         )
+
+    def check_change_filters(self, execution_event: ExecutionBaseEvent):
```

Comment:
I think this function should only be called with `KubernetesAnyChangeEvent`

We can change the event type to `KubernetesAnyChangeEvent`, or we can add an `if isinstance(...)` at the beginning, and return `true` without doing anything on the wrong type

Reply:
Moved to `K8sBaseTrigger`

Reply:
Added an `isinstance` check